package.json

```diff
@@ -40,11 +40,11 @@
     }
   ],
   "devDependencies": {
-    "@types/jasmine": "^2.8.9",
-    "coveralls": "^3.0.2",
-    "istanbul": "^1.1.0-alpha.1",
-    "jasmine": "^3.2.0",
-    "ts-node": "^7.0.1",
-    "typescript": "^3.1.1"
+    "@types/jasmine": "^4.3.1",
+    "coveralls": "^3.1.1",
+    "jasmine": "^3.99.0",
+    "nyc": "^15.1.0",
```

Comment:
It may be easiest to switch to `jest`, then we can get coverage with `jest --coverage`. Would likely require just adjusting the assertions to the jest syntax.

I'm fine with whatever works though :)

Reply:
I'm good with this too. Currently fighting upgrading from istanbul to nyc. If Jest can do it ootb then that works for me.